coco-text/prepare_coco.py

```diff
@@ -0,0 +1,45 @@
+import coco_text
+import numpy as np
+import click
+
+
+@click.command()
+@click.option("--image_path", "-i", default="./images/", help="Directoy where the COCO-2014 images are located.", type=click.Path(exists=True))
+@click.option("--annotations", "-a", default="cocotext.v2.json", help="The json file that contains the annotations.", type=click.Path(exists=True))
+@click.option("--relative/--absolute", default=True, help="Whether the created imagefile should relative or absolute paths.")
+def prepare_coco(image_path, annotations, relative):
+    ct = coco_text.COCO_Text(annotations)
+
+    imgIds = ct.getImgIds(imgIds=ct.train, catIds=[('legibility', 'legible')])
+
+    imgs = ct.loadImgs(imgIds)
+
+    img_ids = [imgs[i]['id'] for i in range(len(imgs))]
+
+    if relative:
+        img_paths = ["./images/" + imgs[i]['file_name'] for i in range(len(imgs))]
+    else:
+        img_paths = [image_path + "/" + imgs[i]['file_name'] for i in range(len(imgs))]
```

Comment:
I'm not sure whether the handling of the `relative` parameter is correct here:

* If I set `relative` to false (and thus want to have absolute paths) but don't change the `image_path` (and thus leave it at the default `./images` value), the resulting image locations are not absolute but in fact still relative – or aren't they?

Futhermore, it seems like the `image_path` variable is not handled correctly when `relative` is set to true:

* If I set `relative` to true (or simply don't set it explicitly) but change the `image_path` to something like `./images-coco/`, the resulting image locations would still point to the default value of the `image_path` (i. e. `./images/`) and not to the path I specified.